oidc-client-graphql tests are failing with NPE

### Describe the bug

https://github.com/quarkusio/quarkus/pull/36375  introduced `oidc-client-grapghql`, some PRs are failing with:

```
2023-10-13T00:41:14.4473397Z Caused by: java.lang.NullPointerException: Cannot invoke "java.util.Map.put(Object, Object)" because "dynamicHeaders" is null
2023-10-13T00:41:14.4475762Z 	at io.quarkus.oidc.client.graphql.runtime.OidcGraphQLClientIntegrationRecorder.lambda$enhanceGraphQLClientConfigurationWithOidc$0(OidcGraphQLClientIntegrationRecorder.java:25)
2023-10-13T00:41:14.4477670Z 	at java.base/java.util.HashMap.forEach(HashMap.java:1421)
2023-10-13T00:41:14.4479641Z 	at io.quarkus.oidc.client.graphql.runtime.OidcGraphQLClientIntegrationRecorder.enhanceGraphQLClientConfigurationWithOidc(OidcGraphQLClientIntegrationRecorder.java:19)
2023-10-13T00:41:14.4482048Z 	at io.quarkus.deployment.steps.OidcGraphQLClientIntegrationProcessor$initialize1724027614.deploy_0(Unknown Source)
2023-10-13T00:41:14.4483872Z 	at io.quarkus.deployment.steps.OidcGraphQLClientIntegrationProcessor$initialize1724027614.deploy(Unknown Source)
```



### Expected behavior

_No response_

### Actual behavior

_No response_

### How to Reproduce?

_No response_

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

_No response_

### GraalVM version (if different from Java)

_No response_

### Quarkus version or git rev

_No response_

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

_No response_

--- a/bom/application/pom.xml
+++ b/bom/application/pom.xml
@@ -55,7 +55,7 @@
         <smallrye-health.version>4.0.4</smallrye-health.version>
         <smallrye-metrics.version>4.0.0</smallrye-metrics.version>
         <smallrye-open-api.version>3.6.2</smallrye-open-api.version>
-        <smallrye-graphql.version>2.5.0</smallrye-graphql.version>
+        <smallrye-graphql.version>2.5.1</smallrye-graphql.version>
         <smallrye-opentracing.version>3.0.3</smallrye-opentracing.version>
         <smallrye-fault-tolerance.version>6.2.6</smallrye-fault-tolerance.version>
         <smallrye-jwt.version>4.3.1</smallrye-jwt.version>

--- a/extensions/oidc-client-graphql/deployment/src/test/java/io/quarkus/oidc/client/graphql/GraphQLClientUsingOidcClientTest.java
+++ b/extensions/oidc-client-graphql/deployment/src/test/java/io/quarkus/oidc/client/graphql/GraphQLClientUsingOidcClientTest.java
@@ -2,7 +2,6 @@ package io.quarkus.oidc.client.graphql;
 
 import static org.hamcrest.Matchers.equalTo;
 
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.RegisterExtension;
 
@@ -12,7 +11,6 @@ import io.quarkus.test.keycloak.server.KeycloakTestResourceLifecycleManager;
 import io.restassured.RestAssured;
 
 @QuarkusTestResource(KeycloakTestResourceLifecycleManager.class)
-@Disabled
 public class GraphQLClientUsingOidcClientTest {
 
     private static final Class<?>[] testClasses = {

